proteobench/modules/dda_quant/module.py

```diff
@@ -370,9 +384,33 @@ def write_intermediate_raw(
         except:
             logging.warning(f"Could not make directory: {path_write}")
 
+        # TODO: save parameters file "locally" together with the raw and intermediate?
         outfile_param = open(os.path.join(path_write, "params.csv"), "w")
         outfile_param.write(str(param_loc.getvalue()))
         outfile_param.close()
 
         input_df.to_csv(os.path.join(path_write, "input_df.csv"))
         result_performance.to_csv(os.path.join(path_write, "result_performance.csv"))
+
+    def load_params_file(
+        self, input_file: str, input_format: str
+    ) -> ProteoBenchParameters:
+        """Method loads parameters from a metadata file depending on its format. TODO: Currently only supports MaxQuant, MSFragger, and Proline"""
+        params = ProteoBenchParameters()
+
+        if input_format == "MaxQuant":
+            params = maxquant.extract_params(input_file)
+        # elif input_format == "AlphaPept":
```

Comment:
also availabe. Would you like to have a dictionary of `extract_params` functions in the module `preteobench.io.params`? Then you could avoid the long block of if/else statements and handle potential key-errros.